devices/common_patches/l4_adc_common.yaml

```
@@ -0,0 +1,166 @@
+_add:
+  # This SVD is missing the ADC_Common peripheral that most other parts with
+  # this ADC contain; consequently it's missing the CSR and CCR and CDR regs
+  # from RM0394.
+  ADC_Common:
+    description: ADC common registers
+    groupName: ADC
+    baseAddress: 0x50040300
+    addressBlock:
+      offset: 0
+      size: 0xC
```

I think this actually needs to be 0x10 because there's 4 reserved bytes in the block; the ADR_CDR register starts at 0xC then continues for 4 bytes.
```suggestion
      size: 0x10
```